src/components/activity-page/activity-page-content.tsx

```diff
@@ -51,11 +51,12 @@ export class ActivityPageContent extends React.PureComponent <IProps> {
   }
 
   public requestInteractiveStates() {
-    const promises = this.props.page.sections.forEach((section: SectionType) =>
-        section.embeddables.map((embeddable: EmbeddableType) =>
-          this.embeddableRefs[embeddable.ref_id]?.current?.requestInteractiveState() || Promise.resolve()
-      )
-    );
+    let promises;
+    this.props.page.sections.forEach((section: SectionType) =>
+      promises = section.embeddables.map((embeddable: EmbeddableType) =>
```

Comment:
This still won't work. The `promises` array will get reset each time through the `forEach` loop. You could initialize the promises to an empty array, and then add the promises to it each time through the loop.